plaso/serializer/json_serializer.py

```diff
@@ -212,14 +212,10 @@ def _ConvertDictToObject(cls, json_dict):
     else:
       raise ValueError('Unsupported class type: {0:s}'.format(class_type))
 
-    container_class = (
+    container_object = (
         containers_manager.AttributeContainersManager.GetAttributeContainer(
             container_type))
-    if not container_class:
-      raise ValueError('Unsupported container type: {0:s}'.format(
-          container_type))
 
-    container_object = container_class()
     supported_attribute_names = container_object.GetAttributeNames()
     for attribute_name, attribute_value in iter(json_dict.items()):
       # Be strict about which attributes to set in non event data attribute
```

Comment:
Per discussion, consider adding a flag here to allow for not raising exceptions on invalid objects if required.

Reply:
I was actually thinking of the merge reader not in the serializer

Reply:
@Onager let chat about adding this flag. So technically this PR is about the storage merge reader. https://github.com/log2timeline/plaso/pull/2897 should prevent parsers writing unsupported attribute container value types going forward. So I don't see a strong argument for putting the flag here. However this might make more sense in the storage reader. I'll have a look and likely draft something in another PR.